cprogramming/LeetCode/q_11/README.md

```
@@ -0,0 +1,117 @@
+## Approach_1 : Brute Force 
+**" In this approach we will take  every pipe/line and calculate the area and then compare it with previous max_area , 
+and if the area is greater than Max_area then store the value of area in math_area.
+"** <br> <br>
+### Algorithm
+1. set a max_area equal to 0
+2. take one pipe from the array and compare it with other pipes.
+3. calculate the area and and compare it with the previous max_area.
+4. repeat step 2, untill all the pipes are compared.
+5. return the max_area
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)